### PR TITLE
Update docs and other assets to refer to new `create-keystone-app`

### PR DESCRIPTION
**Pull request by bladey**
Related PR: https://github.com/keystonejs/create-keystone-next-app/pull/65

### CODE REVIEW

File: examples-next/graphql-api-endpoint/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# keystone-next-app
+# keystone-app
```
Comment (timleslie) [inline examples-next/graphql-api-endpoint/CHANGELOG.md:1]: I don't think we want to change this package.

File: examples-next/graphql-api-endpoint/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "keystone-next-app",
+  "name": "keystone-app",
```
Comment (timleslie) [inline examples-next/graphql-api-endpoint/package.json:2]: I don't think we want to change this package.

**Review by timleslie — CHANGES_REQUESTED** [review]
Just the two reversions and then I think we're good 👍 

Once the CKA churn has died down a bit I'd like us to fully regenerate the cover SVG, but no need to worry about that for now.

File: examples-next/graphql-api-endpoint/CHANGELOG.md (hunk 1 above)
Comment (bladey) [reply examples-next/graphql-api-endpoint/CHANGELOG.md]: Reverted.

File: examples-next/graphql-api-endpoint/package.json (hunk 2 above)
Comment (bladey) [reply examples-next/graphql-api-endpoint/package.json]: Reverted.

**Review by timleslie — APPROVED** [review]
Looks good 👍

**Pull request merged**